x-pack/test/detection_engine_api_integration/security_and_spaces/rule_execution_logic/esql.ts

```diff
@@ -710,7 +710,9 @@ export default ({ getService }: FtrProviderContext) => {
         expect(previewAlerts.length).toBe(150);
       });
 
-      it('should generate alerts when docs overlap execution intervals and alerts number reached max_signals in one of the executions', async () => {
+      // as per https://github.com/elastic/kibana/pull/170034, test is failing on CI and flaky locally
+      // skipping it for now for further investigation
+      it.skip('should generate alerts when docs overlap execution intervals and alerts number reached max_signals in one of the executions', async () => {
```

Comment:
I had to skip this test, as it's flaky locally and failing on CI.
Will be investigated and fixed later

Reply:
Issue to unskip it: https://github.com/elastic/security-team/issues/7928